### PR TITLE
Upgrade Ghost to 1.21.1

### PR DESCRIPTION
**Pull request by sr229**
I also fixed the CI since it fails because it's not exiting to code 0

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -36,6 +36,7 @@ after_script:
     - docker logs blogtest
     - docker stop blogtest
     - docker rm blogtest
+    - exit 0
```
Comment (mmornati) [inline .travis.yml]: Strange to see that we must do this. In the Travis documentation there is

> The exit code of `after_success`, `after_failure`, `after_script` and subsequent stages do not affect the build result. However, if one of these stages times out, the build is marked as a failure.

So I think we have to check why the aster script is not working correctly without putting an Exit 0.
I'm going to check the failure build of your PullRequest

**Review by mmornati — CHANGES_REQUESTED** [review]
I think the build is not working anymore and the add of `exit 0` is just hiding the problem.
capture d ecran 2018-02-10 a 10 21 34

**Comment by sr229** [comment]
@mmornati I think I triaged it on the root cause of failed builds:

- It seems that the reason it fails is because ghost does not permit usage of CLI via root, I had to install su-exec on the builder to force it to install using `node` user

- Permission errors are evident on the build, I think I need to do chowning to be able to get the result intended

**Comment by sr229** [comment]
I also downgraded to `6-alpine` since this is also the image used by official and it seems to work

**Comment by pascalandy** [comment]
LGTM.

I would curious to know why node 8 I crashing.

**Comment by mmornati** [comment]
No idea.... We migrated to node 8 long time ago and we never had problem. I will take a look this evening to the pull requests.

**Review by mmornati — APPROVED** [review]

**Comment by mmornati** [comment]
I'm approving your changes but I will try to switch back to the latest node version.
Good Job and thanks :)

**Comment by pascalandy** [comment]
Will work on it as well

**Pull request merged**